azure-specialized-workloads/oracle/recommendations.yaml

```diff
@@ -0,0 +1,201 @@
+  recommendationResourceType: Specialized.Workload/oracle
+  recommendationMetadataState: Active
+  longDescription:
+    Define a BCDR strategy based on the workload's RPO/RTO requirements.
+    Oracle has set of best practices and reference architectures tailored to meet various workload RPO/RTO requirements. This framework is known as the Oracle Maximum Availability Architecture (MAA).
+    By default, The Oracle Database@Azure solution follows a Silver level reference architecture. The primary database residing in ExaDB-D provides high availability, data protection, elasticity, and scalability benefits, however it is vulnerable to any AZ failure.
+    Gold service level reference architectures with Oracle Database@Azure is recommended. It could be within the same Azure Region or across one or more Azure Regions.
+  potentialBenefits: The cloud MAA architecture achieves data protection and DR.
+  pgVerified: false
+  publishedToLearn: false
+  automationAvailable: no
+  tags: null
+  learnMoreLink:
+    - name: Business continuity and disaster recovery considerations for Oracle Database@Azure
+      url: "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-disaster-recovery-oracle-database-azure"
+    - name: Learn about Oracle Maximum Availability Architecture for Oracle Database@Azure
+      url: "https://docs.oracle.com/en/solutions/oracle-maa-db-at-azure/#GUID-7723E2B1-9588-40BC-88BE-44637B1AF0D9"
+    - name: Oracle Database@Azure Evaluations by Oracle MAA
+      url: "https://docs.oracle.com/en/database/oracle/oracle-database/21/haovw/db-azure1.html#GUID-91572193-DF8E-4D7A-AF65-7A803B89E840"
+
+- description: Implement a backup and restore strategy for Oracle databases to meet your workload requirements
+  aprlGuid: 0583239a-dfb5-44d4-94db-804bfc8e3bd1
+  recommendationTypeId: null
+  recommendationControl: Business Continuity
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/oracle
+  recommendationMetadataState: Active
+  longDescription: |
+    When you configure automatic backup to Autonomous Recovery Service or Object Storage Service in OCI, backup copies provide additional protection. Oracle Recovery Manager (RMAN) validates cloud database backups for any physical corruptions.
+  potentialBenefits: Provide workload data protection.
+  pgVerified: false
+  publishedToLearn: false
+  automationAvailable: no
+  tags: null
+  learnMoreLink:
+    - name: Business continuity and disaster recovery considerations for Oracle Database@Azure
+      url: "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-disaster-recovery-oracle-database-azure"
+    - name: Learn about Oracle Maximum Availability Architecture for Oracle Database@Azure
+      url: "https://docs.oracle.com/en/solutions/oracle-maa-db-at-azure/#GUID-7723E2B1-9588-40BC-88BE-44637B1AF0D9"
+
+- description: Scale up the VM cluster based on the workload requirement
```

Comment:
I wonder if this might be better stated in terms of monitoring the VM cluster and scaling based on alerting on thresholds set via monitoring.

Reply:
Will discuss with the experts